src/bin/lib/pg/snprintf.c

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 1983, 1995, 1996 Eric P. Allman
  * Copyright (c) 1988, 1993
  *	The Regents of the University of California.  All rights reserved.
- * Portions Copyright (c) 1996-2023, PostgreSQL Global Development Group
+ * Portions Copyright (c) 1996-2025, PostgreSQL Global Development Group
```

Comment:
Can you make sure the file has received no other edits in the Postgres source code?

Reply:
The changes are a bit large to include them here, maybe I'll revisit this in a later PR. Dropping for now.